db/migrate/20170430050223_backfill_library_timestamps.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillLibraryTimestamps < ActiveRecord::Migration
```

Comment:
@NuckChorris is this how you want to do it or rather with your custom `update_in_batches` module?